zerver/lib/actions.py

```diff
@@ -6538,14 +6583,19 @@ def email_not_system_bot(email: str) -> None:
 
 class InvitationError(JsonableError):
     code = ErrorCode.INVITATION_FAILED
-    data_fields = ["errors", "sent_invitations"]
+    data_fields = ["errors", "sent_invitations", "license_limit_reached"]
 
     def __init__(
-        self, msg: str, errors: List[Tuple[str, str, bool]], sent_invitations: bool
+        self,
+        msg: str,
+        errors: List[Tuple[str, str, bool]],
+        sent_invitations: bool,
+        license_limit_reached: bool = False,
     ) -> None:
         self._msg: str = msg
         self.errors: List[Tuple[str, str, bool]] = errors
         self.sent_invitations: bool = sent_invitations
+        self.license_limit_reached: bool = license_limit_reached
```

Comment:
I kinda want to move this function to `zerver/lib/exceptions.py` in a prep commit; so that we can move some of the enforcement functions that needs to raise this somewhere in the `corporate/` namespace.